docs/ivys.js

```diff
@@ -78,9 +78,9 @@ Product.prototype.removeTag = function () {
 var ProductCollection = function () {
     "use strict";
     var self = this;
-    self.all_products = {};
-    self.favorite_products = {};
-    self.all_tags = {};
-    self.selected_tags = {};
-    self.selected_products = {};
+    self.all_products = {}; // sku number : 'product name' ?
+    self.favorite_products = {}; // sku number : 'product name' ?
+    self.all_tags = {}; // 'tag-name' : 'tag-name' ?
```

Comment:
I would think that the keys of `all_tags` are the set union of all the tags on the products found in `all_products`.

A `ProductCollection` is a collection of products, so where are the products? The values of `all_products` should probably be another object representing the product.